### PR TITLE
feat(APIApplication): approx guild count and get self application endpoint

### PR DESCRIPTION
**Pull request by almeidx**
**Please describe the changes this PR makes and why it should be merged:**

**If applicable, please reference Discord API Docs PRs or commits that influenced this PR:**
- discord/discord-api-docs#5971

### CODE REVIEW

File: deno/payloads/v10/application.ts [hunk 1]
```diff
@@ -111,6 +112,14 @@ export interface APIApplication {
 	 * which when configured will render the app as a verification method in the guild role verification configuration
 	 */
 	role_connections_verification_url?: string;
+	/**
+	 * An approximate count of the bots guild membership
```
Comment (Jiralite) [inline deno/payloads/v10/application.ts:116]:
```suggestion
	 * An approximate count of the bot's guild membership
```

File: deno/payloads/v9/application.ts [hunk 2]
```diff
@@ -111,6 +112,14 @@ export interface APIApplication {
 	 * which when configured will render the app as a verification method in the guild role verification configuration
 	 */
 	role_connections_verification_url?: string;
+	/**
+	 * An approximate count of the bots guild membership
```
Comment (Jiralite) [inline deno/payloads/v9/application.ts:116]:
```suggestion
	 * An approximate count of the bot's guild membership
```

File: payloads/v10/application.ts [hunk 3]
```diff
@@ -111,6 +112,14 @@ export interface APIApplication {
 	 * which when configured will render the app as a verification method in the guild role verification configuration
 	 */
 	role_connections_verification_url?: string;
+	/**
+	 * An approximate count of the bots guild membership
```
Comment (Jiralite) [inline payloads/v10/application.ts:116]:
```suggestion
	 * An approximate count of the bot's guild membership
```

File: payloads/v9/application.ts [hunk 4]
```diff
@@ -111,6 +112,14 @@ export interface APIApplication {
 	 * which when configured will render the app as a verification method in the guild role verification configuration
 	 */
 	role_connections_verification_url?: string;
+	/**
+	 * An approximate count of the bots guild membership
```
Comment (Jiralite) [inline payloads/v9/application.ts:116]:
```suggestion
	 * An approximate count of the bot's guild membership
```

**Comment by vladfrangu** [comment]
this needs a mega rebase

**Review by Jiralite — APPROVED** [review]

File: deno/rest/v10/mod.ts [hunk 5]
```diff
@@ -887,6 +887,14 @@ export const Routes = {
 	guildOnboarding(guildId: Snowflake) {
 		return `/guilds/${guildId}/onboarding` as const;
 	},
+
+	/**
+	 * Route for:
+	 * - GET `/applications/@me`
+	 */
+	currentApplication() {
```
Comment (vladfrangu) [inline deno/rest/v10/mod.ts:895]: Where's RESTGetCurrentApplicationResult 👀

**Review by vladfrangu — CHANGES_REQUESTED** [review]

**Review by vladfrangu — APPROVED** [review]

**Pull request merged**